packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -77,15 +77,15 @@ augmentFieldsDict(Posts, {
       type: 'Int', 
       resolver: (post, _, context) => {
         if (!post.debate) return 0;
-        return getDialogueResponseCount(post)
+        return getDialogueResponseIds(post).length
       }
     }
   },
   unreadDebateResponseCount: {
     resolveAs: {
       type: 'Int',
       resolver: async (post, _, context) => {
-        if (!post.debate) return 0;
+        if (!post.collabEditorDialogue) return 0;
```

Comment:
This is a backwards-breaking change for existing dialogues.  If we're reusing the same resolver field for both of them, then we need to preserve previous logic for old dialogues.